MediaClippex/DB/MediaClippexDataContext.cs

```diff
@@ -5,8 +5,12 @@ namespace MediaClippex.DB;
 
 public class MediaClippexDataContext : DbContext
 {
+    // ReSharper disable once UnusedMember.Global
     public DbSet<Video> Videos { get; set; } = null!;
 
+    // ReSharper disable once UnusedMember.Global
+    public DbSet<QueuingContent> QueuingVideos { get; set; } = null!;
+
     protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     {
         optionsBuilder.UseSqlite("Data Source=mediaclippex.db"); // SQLite connection string
```

Comment:
rename data source to new file name in order to avoid new update conflict(no migrations made)